integration-tests/src/test/groovy/com/okta/sdk/tests/it/util/ClientProvider.groovy

```
@@ -65,7 +66,12 @@ trait ClientProvider implements IHookable {
         return Strings.hasText(System.getProperty(TestServer.TEST_SERVER_BASE_URL))
     }
 
-    private Client buildClient(String scenarioId = null) {
+    private Client buildClient(String scenarioId = null, String authorizationMode = null) {
+        if (authorizationMode == null) {
+            System.setProperty("okta.client.authorizationMode", AuthorizationMode.NONE.getLabel())
```

my assumption is that `none` should now be the default,  though I'm not sure how this is used, so i could be wrong

---

I still think its safer to set the system property (null and non-null case) whenever _buildClient()_ is invoked because there are several tests that run and there is no guarantee on the order of tests.

---

Maybe, I'm just saying that `NONE` should not be the default setting,  I worry that this potentially hides other problems, lets chat briefly about it Monday.  I'm probably missing something here.

---

You're right actually. This setting shouldn't be done. In the same vein, I realized that the old IT _examples/quickstart/src/test/groovy/com.okta.sdk.examples/QuickstartIT.groovy_ does the same test (OAuth2 flow if the env variables/properties/config is setup for OAuth2 flow). Since this test already covers the same thing we're trying to test again in OAuth2IT, we can get rid of the redundant OAuth2IT test.

---

Also, I'll revert the changes done to this file.